engine/src/test/java/org/camunda/bpm/engine/test/api/history/removaltime/cleanup/HistoryCleanupRemovalTimeTest.java

```diff
@@ -219,44 +219,52 @@ public static void tearDownAfterAll() {
   }
 
   protected final String PROCESS_KEY = "process";
-  protected final BpmnModelInstance PROCESS = Bpmn.createExecutableProcess(PROCESS_KEY)
```

Comment:
🔧 I know this is against the default formatting but I think the indentations are meant to present the BPMN hierarchy and make it easier to read it.

Reply:
I've restored the old formatting.